src/components/autocomplete-grid/autocomplete-grid-button/readme.md

```diff
@@ -0,0 +1,129 @@
+The `sbb-autocomplete-grid-button` component has the same appearance of a [sbb-mini-button](/docs/components-sbb-button-sbb-mini-button--docs),
+but it's only designed to be used within the [sbb-autocomplete-grid-cell](/docs/components-sbb-autocomplete-grid-sbb-autocomplete-grid-cell--docs)
+inside a [sbb-autocomplete-grid](/docs/components-sbb-autocomplete-grid-sbb-autocomplete-grid--docs).
+
+```html
+<sbb-form-field label="Label">
+  <input />
+  <sbb-autocomplete-grid>
+    <sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-option value="1">Option 1</sbb-autocomplete-grid-option>
+      <sbb-autocomplete-grid-cell>
+        <sbb-autocomplete-grid-button icon-name="pen-small"></sbb-autocomplete-grid-button>
+      </sbb-autocomplete-grid-cell>
+    </sbb-autocomplete-grid-row>
+    <sbb-autocomplete-grid-row>
+      <sbb-autocomplete-grid-option value="2">Option 2</sbb-autocomplete-grid-option>
+      <sbb-autocomplete-grid-cell>
+        <sbb-autocomplete-grid-button icon-name="trash-small"></sbb-autocomplete-grid-button>
+      </sbb-autocomplete-grid-cell>
+    </sbb-autocomplete-grid-row>
+  </sbb-autocomplete-grid>
+</sbb-form-field>
+```
+
+## Slots
+
+The component can display a `sbb-icon` using the `iconName` property or via custom content using the `icon` slot.
+
+```html
+<sbb-autocomplete-grid-button icon-name="pen-small"></sbb-autocomplete-grid-button>
+
+<sbb-autocomplete-grid-button>
+  <sbb-icon slot="icon" name="info"></sbb-icon>
+</sbb-autocomplete-grid-button>
+```
+
+## Button properties
+
+The component is internally rendered as a button,
```

Comment:
I think you can't use type, name, form or value here, as the button is not really accessible by keyboard (only virtual). so this docs are feeling misleading here. Do we maybe need to remove all those props? is extending the mini button really a good idea? Maybe needs a discussion.

Reply:
It should be done; now the component extends the ActionBaseElement instead than the minibutton